File: src/pages/exercises/_string-simple-substitution/_codes/ruby/code/print/simple_substitution.print.rb
require_relative "../src/simple-substitution.rb"

# Converting message "lorem ipsum"
puts SimpleSubstitution.substitution('lorem ipsum', 'aegiost', '4361057')
puts 'l0r3m 1p5um'

# Converting letters in numbers

puts SimpleSubstitution.substitution('simple message example', 'aeiou', '43125')
puts 's1mpl3 m3ss4g3 3x4mpl3'

File: src/pages/exercises/_string-simple-substitution/_codes/js/code/simple-substitution.print.js
import { substitution } from "./simple-substitution.js";

// Basic Leet

// converting message to "lorem ipsum"
// https://simple.wikipedia.org/wiki/Leet
// http://www.robertecker.com/hp/research/leet-converter.php?lang=en

console.log(substitution("lorem ipsum", "aegiost", "4361057"));
console.log("l0r3m 1p5um");

// Simple Substitution

// converting to basic leet

console.log(substitution("lorem ipsum", "aegiost", "4361057"));
console.log("l0r3m 1p5um");

// converting to custom leet

console.log(substitution("senha secreta", "aegiost", "!@#$%&*"));
console.log("&@nh! &@cr@*!");
